packages/eslint-config-small-ads/index.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: [
+    "plugin:import/recommended",
+    "plugin:import/typescript",
+    "plugin:react-hooks/recommended",
+    "airbnb-typescript",
+    "prettier",
+    "prettier/react",
+    "prettier/@typescript-eslint",
+  ],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 12,
+    sourceType: "module",
+  },
+  plugins: ["react-hooks", "@typescript-eslint/eslint-plugin"],
+  rules: {
+    "import/prefer-default-export": "off",
+    "import/no-extraneous-dependencies": "off",
+  },
+  settings: {
+    "import/resolver": {
+      node: {
+        extensions: [".js", ".jsx", ".ts", ".tsx"],
+      },
+    },
+  },
+  overrides: [
+    {
+      files: ["*.ts", "*.tsx"],
+      parserOptions: {
+        project: ["tsconfig.json"],
+      },
+    },
+  ],
```

Comment:
You want this to be be in each module we are having and not be living in the eslint package. 

The eslint package is only about eslint configuration we dont wanna be having the tsconfig file in this package but each module should be responsible for their own tsconfig. This way if we wanted the `ui` to have different `tsconfig` from the `fe` package we could be doing that.

Reply:
The same as above, I can see how it is better this way, thanks for pointing out. And for explaining it.